### PR TITLE
Fixed errors from last PR .

### PR DESCRIPTION
**Pull request by mananwason**
@creativepsyco  I have corrected all the things you pointed out and I also used the OSS settings you sent me. Please see and Merge.

### CODE REVIEW

**Comment by leto** [comment]
Nice squashed commit! :+1:

**Comment by mananwason** [comment]
Thanks @leto @creativepsyco helped me a lot.

File: app/src/main/java/org/fossasia/openevent/data/Event.java [hunk 1]
```diff
@@ -14,10 +13,10 @@
     SimpleDateFormat end;
     float latitude;
     float longitude;
-    String location_name;
+    String locationname;
```
Comment (creativepsyco) [inline app/src/main/java/org/fossasia/openevent/data/Event.java]: Should be `locationName`

**Comment by creativepsyco** [comment]
Looks Good one minor change. will merge first. but take note.

File: app/src/main/java/org/fossasia/openevent/data/Event.java (hunk 1 above)
Comment (mananwason) [inline app/src/main/java/org/fossasia/openevent/data/Event.java]: I'll do the Camel casing

**Pull request merged**